src/components/Captures/CaptureTable.js

```diff
@@ -0,0 +1,225 @@
+import React, { useEffect, useState, useContext, createRef } from 'react';
+import {
+  Grid,
+  Table,
+  TableHead,
+  TableBody,
+  TableRow,
+  TableCell,
+  TablePagination,
+  TableSortLabel,
+  Typography,
+} from '@material-ui/core';
+import { getDateTimeStringLocale } from '../../common/locale';
+import { getVerificationStatus } from '../../common/utils';
+import LinkToWebmap from '../common/LinkToWebmap';
+import { CapturesContext } from '../../context/CapturesContext';
+import { SpeciesContext } from '../../context/SpeciesContext';
+import CaptureDetailDialog from '../CaptureDetailDialog';
+import { tokenizationStates } from '../../common/variables';
+import useStyle from './CaptureTable.styles.js';
+
+const columns = [
+  {
+    attr: 'id',
+    label: 'Capture ID',
+  },
+  {
+    attr: 'planterId',
+    label: 'Grower ID',
+  },
+  {
+    attr: 'deviceIdentifier',
+    label: 'Device Identifier',
+    noSort: false,
+  },
+  {
+    attr: 'planterIdentifier',
+    label: 'Planter Identifier',
+    noSort: false,
+  },
+  {
+    attr: 'verificationStatus',
+    label: 'Verification Status',
+    noSort: true,
+  },
+  {
+    attr: 'speciesId',
+    label: 'Species',
+    noSort: true,
+  },
+  {
+    attr: 'tokenId',
+    label: 'Token Status',
+    renderer: (val) =>
+      val ? tokenizationStates.TOKENIZED : tokenizationStates.NOT_TOKENIZED,
+  },
+  {
+    attr: 'timeCreated',
+    label: 'Created',
+    renderer: (val) => getDateTimeStringLocale(val),
+  },
+];
+
+const CaptureTable = () => {
+  const {
+    filter,
+    rowsPerPage,
+    page,
+    order,
+    orderBy,
+    captures,
+    capture,
+    captureCount,
+    setPage,
+    setRowsPerPage,
+    setOrder,
+    setOrderBy,
+    getCaptureAsync,
+  } = useContext(CapturesContext);
+  const speciesContext = useContext(SpeciesContext);
+  const [isDetailsPaneOpen, setIsDetailsPaneOpen] = useState(false);
+  const [speciesState, setSpeciesState] = useState({});
+  const scrollRef = createRef();
+  const classes = useStyle();
+
+  useEffect(() => {
+    formatSpeciesData();
+  }, [filter]);
+
+  const formatSpeciesData = async () => {
+    let species = {};
+    speciesContext.speciesList.map((s) => {
+      species[s.id] = s.name;
+    });
+    setSpeciesState(species);
+  };
+
+  const toggleDrawer = (id) => {
+    getCaptureAsync(id);
+    setIsDetailsPaneOpen(!isDetailsPaneOpen);
+  };
+
+  const createToggleDrawerHandler = (id) => {
+    return () => {
+      toggleDrawer(id);
+    };
+  };
+
+  const closeDrawer = () => {
+    setIsDetailsPaneOpen(false);
+  };
+
+  const handlePageChange = (e, page) => {
+    setPage(page);
+  };
+
+  const handleRowsPerPageChange = (e) => {
+    setRowsPerPage(parseInt(e.target.value));
+  };
+
+  const createSortHandler = (attr) => {
+    return () => {
+      const order = orderBy === attr && order === 'asc' ? 'desc' : 'asc';
```

Comment:
Looks like we have a name clash on `order` and `orderBy` here (local vs props).

Reply:
Sorry, I can't see where the clash is -- they should only be defined via context now, I removed props from the component.  Can you give me more info?